downstream/modules/platform/con-controller-capacity-job-impacts.adoc

```diff
@@ -0,0 +1,13 @@
+[id="controller-capacity-job-impacts"]
+
+= Capacity job impacts
+
+When selecting the capacity, it is important to understand how each job type affects capacity.
+
+The default forks value for Ansible is 5. 
+However, if you set up {ControllerName} to run against fewer systems than that, then the actual concurrency value is lower.
+
+When a job is run in {ControllerName}, an additional 1 is added to the number of forks selected to compensate for the Ansible parent process. 
+
+.Example
+You are running a playbook against 5 systems with a forks value of 5, then the actual forks value from the perspective of Job Impact is 6.
```

Comment:
If you run a playbook against five systems with forks value = 5, then the actual forks value from the Job Impact perspective =6.

Just a suggestion, it's a strange sentence. Numbers less than ten should be written out, but that makes it look odd.